ffi/koptcontext.lua

```diff
@@ -15,6 +15,34 @@ else
     k2pdfopt = ffi.load("libs/libk2pdfopt.so.2")
 end
 
+--[[
```

Comment:
If you add a couple more dashes and move it to the top then ldoc could pick it up? 
```suggestion
--[[--
```